src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.service.impl;
+
+import static core.basesyntax.model.Product.APPLE;
+import static core.basesyntax.model.Product.BANANA;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceImplTest {
+    private static ReportService reportService;
+
+    @BeforeAll
+    static void beforeAll() {
+        reportService = new ReportServiceImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.STORAGE_MAP.clear();
+    }
+
+    @DisplayName("Check report with banana and apple")
+    @Test
+    void createReport_bananaAndApple_ok() {
+        Storage.STORAGE_MAP.put(BANANA.getName(), 152);
+        Storage.STORAGE_MAP.put(APPLE.getName(), 90);
+        String expected = "banana,152"
```

Comment:
It will be more correct introduce expected as List<String> because reportService.createReport() returns List<String>. It isn't needed to make additional convertation  result to the String.
```suggestion
        List<String> expected = "banana,152"
```
This is true for all similar test cases.